### DISCUSSION TITLE
搜索功能下线了吗

### DISCUSSION THREAD

**Discussion by jing5460** [AlistGo/alist#950] [Q&A]
搜索栏功能是不是取消了？

**Comment by xhofe** [comment]
v2从来没有过。

**Reply by Cyiming** [reply]
> v2从来没有过。

我记得有 在右上角 默认不开启

**Reply by LWBobo** [reply]
> v2从来没有过。

v3版本有嘛？我咋也没找到啊...

**Reply by XiaDao** [reply]
> > v2从来没有过。
> 
> v3版本有嘛？我咋也没找到啊...

你升级最新的3.6就有了

**Reply by LWBobo** [reply]
> > > v2从来没有过。
> > 
> > 
> > v3版本有嘛？我咋也没找到啊...
> 
> 你升级最新的3.6就有了

好的，我装的是3.5.1，我升级一下试试
